saleor/graphql/views.py

```diff
@@ -204,7 +213,9 @@ def handle_query(self, request: HttpRequest) -> JsonResponse:
                     span.set_attribute(f"ip_{additional_ip_header}", request_ips[:100])
 
             response = self._handle_query(request)
-            span.set_attribute(SpanAttributes.HTTP_STATUS_CODE, response.status_code)
+            tracer.get_current_span().set_attribute(
```

Comment:
Isn't `tracer.get_current_span()` the same as `span` here?

Reply:
You are right. It is just an example of using `tracer.get_current_span()` which I forgot to remove